driver/razerkbd_driver.c

```diff
@@ -667,6 +668,8 @@ static ssize_t razer_attr_read_device_type(struct device *dev, struct device_att
         device_type = "Razer Blade 15 Advanced (Mid 2021)\n";
         break;
 
+    case USB_DEVICE_ID_RAZER_BLADE_15_BASE_EARLY_2021:
+        device_type = "Razer Blade 15 Base (Early 2021)\n";
```

Comment:
The indentation is a bit wonky, and it needs a break statement, otherwise it'll fall through to the next item.

Reply:
Added the break; and a new line - wasn't sure what you meant by indentation as my new lines are same indents as lines around it? Let me know if there is still changes needed here...

Reply:
Changes made based on feedback.